hifitime.pyi

```diff
@@ -0,0 +1,81 @@
+# This file is automatically generated by pyo3_stub_gen
```

Comment:
It seems that the stub generator could not find any of the functions in any of the classes, so as-is this new `pyi` file won't help IDE users in autocompleting.

Reply:
Yes. So maybe we can turn to another way to do this, like dora's?

Reply:
I like that dora's approach uses the package introspection, but I could not seem to get it to work correctly when I tried a few weeks (months?) ago. It kept on complaining that one of the return types was not defined although I thought I had. Feel free to give it a try of course!